bob-apps/bin/bobp.rs

```diff
@@ -640,6 +716,27 @@ fn print_periodic_stat(
 
             get_speed_values.push(get_spd);
         }
+        if print_delete {
+            if !first_line {
+                print!("{:>6}", ' ');
+            } else {
+                first_line = false;
+            }
+
+            let d_delete = delete_count.get_diff();
```

Comment:
This specific for operation statistics updating logic can be moved to a specific struct (e.g. this one to `DeleteStatistics`)